stable/yugaware/values.yaml

```diff
@@ -62,8 +62,12 @@ yugaware:
       cpu: 2
       memory: 4Gi
   enableProxyMetricsAuth: true
+  ## List of additional alowed CORS origins in case of complex rev-proxy
+  additionAllowedCorsOrigins: []
```

Comment:
Can we add the teleport proxied platform addrs to this list? This was the issue I was running into yesterday

Reply:
Yes, i;m working on this

Reply:
we shouldn't hardcode the value here, correct? instead, we'd want to add it as a values override when deploying the platform when teleport proxy addrs are required as an allowed origin

Reply:
Right.

Reply:
This patch is in "Ready for review" state. So no changes expected, but someone asked for it.

Reply:
So I'm working on this in Cloud infra.